docs/_files/code/plots_grids.py

```diff
@@ -0,0 +1,285 @@
+import matplotlib.colors as mcolors
```

Comment:
A docstring to this module would be helpful. As far as I understood it is generating figures of the doc, right?